ivy_tests/test_ivy/test_frontends/test_jax/test_numpy/test_indexing.py

```diff
@@ -12,7 +12,7 @@
     _helper_r_,
     _helper_c_,
 )
-import ivy.functional.frontends.jax.numpy as jnp_frontend
+import ivy.functional.frontends.jax.numpy as jnp
```

Comment:
Yeah, since the test helper that caused the error checks strings for the keyword 'frontend', this being ast extracted caused an error. Is there an alternative?

Reply:
I'll change this to jnp_frontend for the test file.

It being imported as 'jnp_frontend' in the actual frontend functions file was causing an error.